crane/container.go

```diff
@@ -69,15 +69,15 @@ type RunParameters struct {
 	RawCapAdd       []string    `json:"cap-add" yaml:"cap-add"`
 	RawCapDrop      []string    `json:"cap-drop" yaml:"cap-drop"`
 	RawCgroupParent string      `json:"cgroup-parent" yaml:"cgroup-parent"`
-	CpuPeriod       int         `json:"cpu-period" yaml:"cpu-period"`
```

Comment:
Oh. I must have confused it then with something else. I guess it's better to follow the guideline then, so leave it in.

Thanks for pointing me to the document.